src/viking.js

```diff
@@ -1,11 +1,102 @@
 // Soldier
-class Soldier {}
+class Soldier {
+    constructor(health, strength) {
+        this.health = health;
+        this.strength = strength;
+    }
+
+    attack() {
+        return this.strength = this.strength;
+      }
+
+    receiveDamage(damage) {
+        this.health = this.health - damage;
+      }
+
+}
 
 // Viking
-class Viking {}
+class Viking extends Soldier {
+    constructor(name, health, strength)  {
+        super(health, strength);
+        this.name = name;
+    }
+    
+    receiveDamage(damage) {
+        this.health = this.health - damage;
+        if (this.health > 0) {
```

Comment:
could use a ternary here instead of the if/else returns. 
return this.health ? `${this.name} has received ${damage} points of damage` : `${this.name} has died in act of combat`

both work, just so you know :)